### ISSUE TITLE
[BUG]:get exception on openAudioSession

### ISSUE THREAD

**Issue by primemb** [Canardoux/tau#626]
## Flutter Sound Version : ^7.6.4+3

- **FULL** or **LITE** flavor ? LITE

- **Important**: Result of the command : ```flutter pub deps | grep flutter_sound```

```
$ flutter pub deps | grep flutter_sound
|-- flutter_sound_lite 7.6.4+3
|   |-- flutter_sound_platform_interface 7.6.4+3
|   |-- flutter_sound_web 7.6.4+3
|   |   |-- flutter_sound_platform_interface...

```

my init code :
```
  @override
  void initState() {
    _mPlayer.openAudioSession().then((value) {
      setState(() {
        _mPlayerIsInited = true;
      });
    });
    super.initState();
  }
```

error on init : 

```
I/flutter (29899): FS:---> openAudioSession
I/flutter (29899): Resetting flutter_sound Player Plugin
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): Failed to handle method call
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): java.lang.RuntimeException
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at com.dooboolab.fluttersound.FlutterSoundManager.getSession(FlutterSoundManager.java:85)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at com.dooboolab.fluttersound.FlutterSoundPlayerManager.onMethodCall(FlutterSoundPlayerManager.java:83)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:692)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at android.os.MessageQueue.next(MessageQueue.java:323)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at android.os.Looper.loop(Looper.java:136)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at android.app.ActivityThread.main(ActivityThread.java:6077)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:866)
E/MethodChannel#com.dooboolab.flutter_sound_player(29899): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:756)
E/flutter (29899): [ERROR:flutter/lib/ui/ui_dart_state.cc(177)] Unhandled Exception: PlatformException(error, null, null, java.lang.RuntimeException
E/flutter (29899): 	at com.dooboolab.fluttersound.FlutterSoundManager.getSession(FlutterSoundManager.java:85)
E/flutter (29899): 	at com.dooboolab.fluttersound.FlutterSoundPlayerManager.onMethodCall(FlutterSoundPlayerManager.java:83)
E/flutter (29899): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
E/flutter (29899): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
E/flutter (29899): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:692)
E/flutter (29899): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/flutter (29899): 	at android.os.MessageQueue.next(MessageQueue.java:323)
E/flutter (29899): 	at android.os.Looper.loop(Looper.java:136)
E/flutter (29899): 	at android.app.ActivityThread.main(ActivityThread.java:6077)
E/flutter (29899): 	at java.lang.reflect.Method.invoke(Native Method)
E/flutter (29899): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:866)
E/flutter (29899): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:756)
E/flutter (29899): )
[38;5;244mE/flutter (29899): #0      StandardMethodCodec.decodeEnvelope[39;49m
 package:flutter/…/services/message_codecs.dart
[38;5;244mE/flutter (29899): #1      MethodChannel._invokeMethod[39;49m
 package:flutter/…/services/platform_channel.dart
E/flutter (29899): <asynchronous suspension>
[38;5;244mE/flutter (29899): #2      MethodChannel.invokeMethod[39;49m
 package:flutter/…/services/platform_channel.dart
[38;5;244mE/flutter (29899): #3      MethodChannelFlutterSoundPlayer.invokeMethod[39;49m
 package:flutter_sound_platform_interface/method_channel_flutter_sound_player.dart
[38;5;244mE/flutter (29899): #4      MethodChannelFlutterSoundPlayer.openPlayer[39;49m
 package:flutter_sound_platform_interface/method_channel_flutter_sound_player.dart
[38;5;244mE/flutter (29899): #5      FlutterSoundPlayer._openAudioSession[39;49m
 package:flutter_sound_lite/public/flutter_sound_player.dart
E/flutter (29899): <asynchronous suspension>
[38;5;244mE/flutter (29899): #6      FlutterSoundPlayer.openAudioSession.<anonymous closure>[39;49m
 package:flutter_sound_lite/public/flutter_sound_player.dart
[38;5;244mE/flutter (29899): #7      BasicLock.synchronized[39;49m
 package:synchronized/src/basic_lock.dart
[38;5;244mE/flutter (29899): #8      FlutterSoundPlayer.openAudioSession[39;49m
 package:flutter_sound_lite/public/flutter_sound_player.dart
[38;5;248mE/flutter (29899): #9      _VisitListState.initState[39;49m
 package:doctor_app_newsdk/screens/visit_list.dart
[38;5;244mE/flutter (29899): #10     StatefulElement._firstBuild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #11     ComponentElement.mount[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #12     Element.inflateWidget[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #13     Element.updateChild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #14     SingleChildRenderObjectElement.mount[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #15     Element.inflateWidget[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #16     Element.updateChild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #17     ComponentElement.performRebuild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #18     Element.rebuild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #19     ComponentElement._firstBuild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #20     ComponentElement.mount[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #21     Element.inflateWidget[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #22     Element.updateChild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #23     SingleChildRenderObjectElement.mount[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #24     Element.inflateWidget[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #25     Element.updateChild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #26     SingleChildRenderObjectElement.mount[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #27     Element.inflateWidget[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #28     Element.updateChild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #29     ComponentElement.performRebuild[39;49m
 package:flutter/…/widgets/framework.dart
[38;5;244mE/flutter (29899): #30     StatefulElement.performRebuild[39;49m
 package:flutter/…/widgets/framework.dart
E/flutter (29899): #31     Element.rebuild (package:flutter/src/widget


```

**Comment by Larpoux** [comment]
I just fixed something in 7.6.5.

Can you try if better ?
If not better, could you give me also the log lines before openAudioSession() ?

**Comment by primemb** [comment]
yes, it fixed thank you. 💯
